pkg/scheduler/framework/types.go

```diff
@@ -78,6 +78,62 @@ const (
 	WildCard              GVK = "*"
 )
 
+type ClusterEventWithHint struct {
```

Comment:
The name look odd an too specific; why are we adding a new type? why not place QueueingHintFn inside ClusterEvent?

Reply:
I guess the question is: do we ever use ClusterEvent on its own?

Reply:
I think in some logic like MoveAllToActiveOrBackoffQueue(), ClusterEvent w/o hintFn is used.